src/i965_drv_video.c

```diff
@@ -6409,21 +6425,22 @@ i965_QuerySurfaceAttributes(VADriverContextP ctx,
     i++;
 
     if (obj_config->entrypoint == VAEntrypointEncSlice ||
-        obj_config->entrypoint == VAEntrypointEncSliceLP) {
+        obj_config->entrypoint == VAEntrypointEncSliceLP ||
+        obj_config->entrypoint == VAEntrypointFEI) {
         attribs[i].type = VASurfaceAttribMinWidth;
         attribs[i].value.type = VAGenericValueTypeInteger;
         attribs[i].flags = VA_SURFACE_ATTRIB_GETTABLE;
-        attribs[i].value.value.i = 32;
+        attribs[i].value.value.i = I965_MIN_CODEC_ENC_RESOLUTION_WIDTH;
         if (obj_config->profile == VAProfileJPEGBaseline)
-            attribs[i].value.value.i = 16;
+            attribs[i].value.value.i = 1;
```

Comment:
What was the original reason for using `16`?

Reply:
it is the limitation in the closed source code. and there is no limitation here.